lib/v2/hackernews/maintainer.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    '/:section?/:type?': ['nczitzk'],
+    '/:section?/:type?/:user?': ['nczitzk'],
```

Comment:
```suggestion
    '/:section?/:type?/:user?': ['nczitzk', 'xie-dongping'],
```
Please also put your name in it.

Reply:
Done